src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/deployment_map.py

```diff
@@ -101,4 +100,4 @@ def _process_dir(self, path):
                     self._read(filename)
                 )
             else:
-                LOGGER.warning("%s is not a directory and doesn't end in.yml", filename)
+                LOGGER.warning(f"{filename} is not a directory and doesn't end in.yml")
```

Comment:
Could you please revert this change? The logger has a capability to insert strings only when the logging level is enabled. So the filename will not be inserted if logging is disabled. In this case probably all loggers would pick up on warnings. However, I would like to keep those consistent if possible.

```suggestion
                LOGGER.warning("%s is not a directory and doesn't hold the .yml suffix", filename)
```